setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="VoA",
+    version="0.0.1",
+    author="Mateusz Przyborowski, Krzysztof Suwada",
+    description="Tools proposed in the paper 'Validation of Association'",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    url="https://arxiv.org/pdf/1904.06519",
```

Comment:
I think this should point to this github, and link to paper should be in readme.